Client/Configuration/AdHoc.xcconfig

```diff
@@ -2,3 +2,6 @@ BUNDLE_DISPLAY_NAME = $(TARGET_NAME) β
 
 // Bundle Identifier
 APP_BUNDLE_ID = $(PRODUCT_BUNDLE_IDENTIFIER).nightly
+
+// URL Scheme
+CLIQZ_URL_SCHEME = cliqznightly
```

Comment:
What about Ghostery? The value should probably be in the format of `APP_BUNDLE_ID = $(PRODUCT_BUNDLE_IDENTIFIER).nightly`

Reply:
Done.